### DISCUSSION TITLE
about prefer-template rules

### DISCUSSION THREAD

**Discussion by chuhangyu** [eslint/eslint#14654] [Q&A]
my codes are like this
`month = month > 9 ? month : '0' + month;`

after eslint fix
` month = month > 9 ? month : ``0${  month}``;`

I want to know how to avoid the space in the brace

**Comment by mdjermanovic** [comment]
Hi @chuhangyu,

You can enable the [template-curly-spacing](https://eslint.org/docs/rules/template-curly-spacing) rule, which will then remove spaces in the fixed code.

[Online demo with both rules](https://eslint.org/demo#eyJ0ZXh0IjoiLyogZXNsaW50IHByZWZlci10ZW1wbGF0ZTogXCJlcnJvclwiICovXG4vKiBlc2xpbnQgdGVtcGxhdGUtY3VybHktc3BhY2luZzogXCJlcnJvclwiICovXG5cbm1vbnRoID0gbW9udGggPiA5ID8gbW9udGggOiAnMCcgKyBtb250aDsiLCJvcHRpb25zIjp7InBhcnNlck9wdGlvbnMiOnsiZWNtYVZlcnNpb24iOjEyLCJzb3VyY2VUeXBlIjoic2NyaXB0IiwiZWNtYUZlYXR1cmVzIjp7fX0sInJ1bGVzIjp7fSwiZW52Ijp7fX19) (see the Fixed Code tab).
